### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by kevinrue** [carpentries-incubator/bioc-project#47] [General]
Please use this thread to introduce yourselves!

Everyone is welcome!

**Comment by kevinrue** [comment]
Hello,

My name is Kevin Rue-Albrecht.
I am a post-doctoral researcher and biomedical data science trainer at the University of Oxford.

I've been a user of Bioconductor packages since my the start of my Ph.D. degree in 2011.
I developed my first Bioconductor package in 2013 (GOexpress), and continued developing more since (TVTB, iSEE, iSEEu, iSEEhex, iSEEhub, velociraptor).

After attending some Bioconductor conferences in Europe and North America, I've been helping as a co-organiser of Bioconductor events, both in person and online. I particularly enjoy constructive discussions and the exchange of ideas with other community members.

I am also a member of the Bioconductor Community Advisory Board (CAB) and several working groups in the Bioconductor community, namely:

- The education committee
- New website working group
- New developer and mentorship working group

I've also enjoy working in Python, C#, JavaScript, and other programming languages over the years.

At the time of writing, I've written most of the contents in this draft lesson for The Carpentries. I very much welcome feedback and contributions from other community members on the existing contents, as well as suggestions on episodes to add to this lesson in the future.

I very much look forward to working with anyone willing to join the effort!

**Comment by kozo2** [comment]
Hello,

My name is Kozo Nishida.
I am an research software engineer in Tokyo University of Agriculture and Technology (TUAT).

I developed my first Bioconductor package in 2020 (transomics2cytoscape), and continued developing more since (AHPathbankDbs, AHWikipathwaysDbs).

After attending some Bioconductor related meetups in Japan, I've been helping as a co-organiser of Bioconductor online events.
I particularly enjoy the exchange of ideas with other community members.

I've also enjoy working in Python, C#, and other programming languages.
I'm one of the organizers of the PyData meetup in Japan.
http://osaka.pydata.org/
Actually I moved from Osaka to Tokyo this year, but I'm still an organizer of PyData Osaka.
Because the pandemic control policy is still strict in Japan and almost all meetups are still online.
Also, I have been involved in the development of [MS-DIAL](https://pubmed.ncbi.nlm.nih.gov/25938372/) since this year, and I usually write code in C#.
(MS-DIAL is written in C#.)

I am also a Carpentries instructor.
https://carpentries.org/instructors/
I also translate Carpentries teaching materials into Japanese.
https://github.com/swcarpentry-ja
And as a member of [the Bioconductor multilingual working group](http://workinggroups.bioconductor.org/currently-active-working-groups-committees.html#multilingual),
I also hold [the monthly Bioconductor translation-a-thon](https://www.meetup.com/boston-r-bioconductor-for-genomics/events/287802282/).
(We use a translation platform called Crowdin for the translation. https://bioconductor.crowdin.com/ .)

I'm looking forward to joining in the original course effort instead of the translation.
